cli/src/config.rs

```diff
@@ -259,11 +259,31 @@ pub struct ConfigEnv {
 impl ConfigEnv {
     /// Initializes configuration loader based on environment variables.
     pub fn from_environment() -> Self {
+        #[cfg(not(target_os = "macos"))]
```

Comment:
nit: I think it's generally preferred to do `if cfg!(target_os = "macos")` so that it's compile-tested on all platforms.